### ISSUE TITLE
Cross_entropy的文档描述不全

### ISSUE THREAD

**Issue by btobab** [PaddlePaddle/Paddle#30958]
paddle的cross_entropy函数已经包含了softmax激活，所以在模型配置时不需要加上softmax层。
今天我上淘宝求助了半天才搞清楚这个问题。。。

**Comment by joey12300** [comment]
> paddle的cross_entropy函数已经包含了softmax激活，所以在模型配置时不需要加上softmax层。
> 今天我上淘宝求助了半天才搞清楚这个问题。。。

您好，请问是参考哪里的文档呢？目前该函数2.0版本的文档：https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/layer/loss/CrossEntropyLoss_cn.html#crossentropyloss 这份文档的第一句描述：`该OP计算输入input和标签label间的交叉熵损失 ，它结合了 LogSoftmax 和 NLLLoss 的OP计算，可用于训练一个 n 类分类器。`

**Comment by btobab** [comment]
我参考的是2.0的paddle.nn.functional.cross_entropy函数

**Comment by geoyee** [comment]
请问为什么在nn的CrossEntropy中没有soft_label，在nn.functional的cross_entropy中就有

**Comment by btobab** [comment]
> 请问为什么在nn的CrossEntropy中没有soft_label，在nn.functional的cross_entropy中就有

paddle2.0的nn的CrossEntropyLoss包含了softmax激活，nn.functional的cross_entropy也包含了softmax丫
